sections/builder.js

```
@@ -1655,6 +1655,98 @@ export default [
       }
     ]
   },
+  {
+    type: "method",
+    method: "onConflict",
+    example: "insert(..).onConflict(column) / insert(..).onConflict([column1, column2, ...])",
+    description: "Utlized by PostgreSQL and Sqlite databases. When chained onto an insert query, it specifies the columns to be used in `ON CONFLICT` clause. A call to .onConflict should always be followed by either .ignore or .update (otherwise it does nothing). In MySQL this method is noop, and .ignore and .merge can be used without .onConflict.",
+    children: [
+      {
+        type: "text",
+        content: "Note: The column(s) specified by this method must have a UNIQUE index on them or the INSERT..ON CONFLICT query will fail to execute. When specifying multiple columns, they must have composite UNIQUE index."
+      },
+    ]
+  },
+  {
+    type: "method",
+    method: "ignore",
+    example: "insert(..).onConflict(..).ignore()",
+    description: "Implemented for the PostgreSQL, MySQL, and Sqlite databases. Modifies an insert query, and causes it silently dropped without error if a conflict occurs. Uses INSERT IGNORE in MySQL, and adds an ON CONFLICT (columns) DO NOTHING clause to the insert statement in PostgreSQL and Sqlite.",
```

`and causes it to be silently dropped without an error`?